package.json

```
@@ -121,7 +121,9 @@
 		"schoolExport": "node ./scripts/schoolExport.js",
 		"schoolImport": "node ./scripts/schoolImport.js",
 		"generate-client:authorization": "node ./scripts/generate-client.js -u 'http://localhost:3030/api/v3/docs-json/' -p 'apps/server/src/infra/authorization-client/authorization-api-client' -c 'openapitools-config.json' -f 'operationId:AuthorizationReferenceController_authorizeByReference'",
-		"generate-client:etherpad": "node ./scripts/generate-client.js -u 'http://localhost:9001/api/openapi.json' -p 'apps/server/src/infra/etherpad-client/etherpad-api-client' -c 'openapitools-config.json'"
+		"generate-client:etherpad": "node ./scripts/generate-client.js -u 'http://localhost:9001/api/openapi.json' -p 'apps/server/src/infra/etherpad-client/etherpad-api-client' -c 'openapitools-config.json'",
+		"pregenerate-client:tsp-api": "rimraf ./apps/server/src/infra/tsp-client/generated",
+		"generate-client:tsp-api": "openapi-generator-cli generate -c ./openapitools.json --generator-key tsp-api"
```

Why two separate commands, couldn't we combine them into one command like this: ```suggestion
		"regenerate-client:tsp-api": "rimraf ./apps/server/src/infra/tsp-client/generated && openapi-generator-cli generate -c ./openapitools.json --generator-key tsp-api"
```

---

'pregenerate...' is executed automatically when you run the 'generate...' command. 'post' and 'pre' are keywords in node. This was already used in our 'postinstall', for example.